### DISCUSSION TITLE
Gruvbox theme.

### DISCUSSION THREAD

**Discussion by kenthinson** [FreeTubeApp/FreeTube#6427] [General]
Hello. Today I installed FreeTube and set it up with the Gruvbox theme. Was very excited to see that color scheme supported. However after loading it up I noticed it was a bit brighter then the rest of my apps. All my other apps are using popular themes from the AUR. 

Example:
![2024-12-22-005059_hyprshot-Before](https://github.com/user-attachments/assets/b25f3aa9-5a03-4655-85bd-80bc08a80c02)

I looked up the Gruvbox color scheme and I figured out that the color bg2 was selected. While most other public themese use bg1. I know it's a slight difference when looking at the color key. However when looking at the apps next to each other it makes the window seem quite a bit brighter. Especially at night. 
![687474703a2f2f692e696d6775722e636f6d2f776136363678672e706e67](https://github.com/user-attachments/assets/70b8faa2-41f6-4e19-ad1b-6d755c8a3f4b)

I compiled my own version today with the change from bg2 to bg1. here is a screen shot to show how it matches most other public themes better 
![2024-12-22-011846_hyprshot-After](https://github.com/user-attachments/assets/9d6bf44f-d479-4a77-9827-50860c067a6f)

So now the discussion. Does anyone else care about this besides me enough to get this into the mainline project? What's the process like if so? Do the devs want a pull request or do they just want to go in and change it because it's so quick / easy?

Here is the code I changed in themes.css inside .gruvboxDark

```
  --card-bg-color: #3c3836;
  --secondary-card-bg-color: #3c3836;
  --side-nav-color: #3c3836;
  --side-nav-active-color: #3c3836;
```

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
It could be possible that our implementation of certain themes differs from the official color scheme. Allot of themes are using colors that look "good" but none of them consider Color Contrast. We always follow the Web Content Accessibility Guidelines (WCAG), which are a series of recommendations for making the web more accessible. Regarding colors, the standard defines two levels of contrast ratio: AA (minimum contrast) and AAA (enhanced contrast). The level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text (at least 18pt) or bold text. The level AAA requires a contrast ratio of at least 7:1 for normal text and 4.5:1 for large text or bold text.

See my [review comment](https://github.com/FreeTubeApp/FreeTube/pull/5887#pullrequestreview-2388294700) about it on the Gruvbox theme PR.

I'd recommend reading the whole discussion on that PR to gain more insight in what has changed and why

**Reply by kenthinson** [reply]
@efb4f5ff-1298-471a-8973-3d47447115dc Thanks for the info about the feature thread! I will leave a comment for the author and let him think about how to proceed.
